diff --git a/plugin/src/Caelestia/Config/launcherconfig.hpp b/plugin/src/Caelestia/Config/launcherconfig.hpp
--- a/plugin/src/Caelestia/Config/launcherconfig.hpp
+++ b/plugin/src/Caelestia/Config/launcherconfig.hpp
@@ -31,6 +31,7 @@ class LauncherConfig : public settings::ObjectNode {
     CONFIG_PROPERTY(bool, showOnHover, false)
     CONFIG_PROPERTY(int, maxShown, 7)
     CONFIG_PROPERTY(int, maxWallpapers, 9)
+    CONFIG_PROPERTY(bool, colorizeIcons, false)
     CONFIG_GLOBAL_PROPERTY(QString, specialPrefix, u"@"_s)
     CONFIG_GLOBAL_PROPERTY(QString, actionPrefix, u">"_s)
     CONFIG_GLOBAL_PROPERTY(bool, enableDangerousActions, false)
